src/platforms/apple/common/dsym.mdx

```diff
@@ -24,9 +24,13 @@ _NOTE_: This feature will be available as an early access preview soon.
 
 When you use the App Store Connect integration, Sentry will automatically discover and fetch dSYM files for Bitcode builds as they become available on App Store Connect. These dSYM files however contain obfuscated symbol names and paths. An additional BCSymbolMap file needs to be uploaded to provide properly symbolicated crash reports. These BCSymbolMap files can be uploaded using either [sentry-cli](#symbolmap-sentrycli) or [Fastlane](#symbolmap-fastlane).
 
-To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect as a new "Custom Repository". Note that setting up the integration will currently result in two identical audit entries being created.
+To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect as a new "Custom Repository".
```

Comment:
We use quotation marks in the text to quote UI text, so it has to be an exact match. You can also use the phrasing you had before and remove the capital letters and the quotation marks (so: a new custom repository).
```suggestion
To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect in the "Custom Repositories" section.
```

Reply:
i've also added quotes around "App Store Connect" since it refers to a specific item in the add repositories dropdown.